supporting_content/columns/skumetername.md

```diff
@@ -0,0 +1,44 @@
+# Column: SkuMeterName
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                 | Column                             |
+|-----------|--------------------------|------------------------------------|
+| AWS       | CUR                      | line_item_usage_type               |
+| GCP       | Big Query Billing Export | service.description                |
+| Microsoft | Cost details             | MeterName                          |
+| OCI       | Cost reports             | TBD                                |
+
+## Background
+
+Cloud Services, such as managed databases, virtual machines for compute, and object storage, have billing models in which the usage of that service is metered and then charged based on multiple parameters. For example, Amazon S3 charges customers for Storage, API requests, Data transfer, Encryption, Object management, Replication, and custom API requests, and ALL of these capabilities are metered and billed under the service name “Amazon S3”.
+
+Cloud providers use different SKUs to differentiate between these different types of usage of a given service, but the SKU is a GUID that does not have any semantic meaning to a FinOps practitioner. This is problematic because as a FinOps practitioner, I want to be able to understand what the breakdown is of how I am being charged for a particular service so that I can use this information to report what the primary cost drivers are for a particular service and so that I can make optimization decisions. For example, I might want to understand what my data transfer costs within all of my Amazon S3 charges so that I can recommend my development team to change their storage access patterns in order to achieve cost savings.
+
+Research across cloud providers suggests that there are 4 main categories of attributes that contribute to a SKU. They are:
+Product: What is the public name of the thing (eg the resource or service) being sold?
+Region: Where is the thing being sold?
+Function: What is the capability being measured and sold?
+Size: How big is the thing (eg the resource or service) being sold (if applicable)?
+
+This column addresses the "Function" attribute of a SKU by making it a first class part of the FOCUS schema.
+
+## Problem
+
+Today, FOCUS does not have any columns that enable FinOps practitioners to understand the meaning of different SKUs that appear and group them (such as with SQL GROUP BY) in order to better understand the main contributors to the cost of a given service.
+
+As a practitioner, I want to be able to write a SQL query that easily breaks down my charges for a given service so that I can understand how each kind of "Function" is contributing to the overall cost of a service.
+
+## Solution Discussion
```

Comment:
Would it make sense to add a naming discussion to this as well? I'd hate for people to look at this and assume Microsoft pushed this name. I for one have concerns with the name based on Microsoft's historical usage of it. I'm not trying to push back, but it would be nice to have some overview of the names we discussed and why we landed on this one.

Reply:
Yeah let's put together another vote for the name with some of the ideas we discussed on slack

Reply:
TF-2 voted for SkuMeter today (9/11/24)